test/PwmRotationTest.cpp

```diff
@@ -0,0 +1,263 @@
+/**
+ * @file   PwmRotationTest.cpp
+ * @brief  PwmRotationクラスのテスト
+ * @author Negimarge
+ */
+
+#include "PwmRotation.h"
+#include <gtest/gtest.h>
+#include <cmath>
+#include <gtest/internal/gtest-port.h>
+
+using namespace std;
+
+namespace etrobocon2024_test {
+
+  // @see https://github.com/KatLab-MiyazakiUniv/etrobocon2022/blob/main/docs/odometry.md
+  constexpr double TRANSFORM = 2.0 * RADIUS / TREAD;  // 回頭角度を求める式の係数
+
+  // 右回頭のテスト
+  TEST(PwmRotationTest, runRight)
+  {
+    Measurer measurer;
+    int angle = 90;
+    int pwm = 100;
+    bool isClockwise = true;
+    PwmRotation PwmRotation(angle, pwm, isClockwise);
+
+    double expected = angle;  // 指定した回頭角度を期待値とする
+
+    // 一回のsetPWM()でダミーのモータカウントに加算される値はpwm * 0.05
+    double error = pwm * 0.05 * TRANSFORM;  // 許容誤差[deg]
+
+    // 回頭前のモータカウント
+    int initialRightMotorCount = measurer.getRightCount();
+    int initialLeftMotorCount = measurer.getLeftCount();
+
+    testing::internal::CaptureStdout();                      // 標準出力キャプチャ開始
+    PwmRotation.run();                                       // 右回頭を実行
+    string output = testing::internal::GetCapturedStdout();  // キャプチャ終了
+
+    // 回頭後に各モータが回転した角度
+    int rightMotorCount = abs(measurer.getRightCount() - initialRightMotorCount);
+    int leftMotorCount = abs(measurer.getLeftCount() - initialLeftMotorCount);
+    double actual = ((rightMotorCount + leftMotorCount) * TRANSFORM) / 2;
+
+    EXPECT_LE(expected, actual);
+    EXPECT_GE(expected + error, actual);
+  }
+
+  // 左回頭のテスト
+  TEST(PwmRotationTest, runLeft)
+  {
+    Controller controller;
+    controller.resetRightMotorPwm();
+    controller.resetLeftMotorPwm();
+
+    Measurer measurer;
+    int angle = 180;
+    int pwm = 100;
+    bool isClockwise = false;
+    PwmRotation PwmRotation(angle, pwm, isClockwise);
+
+    double expected = angle;  // 指定した回頭角度を期待値とする
+
+    // 一回のsetPWM()でダミーのモータカウントに加算される値はpwm * 0.05
+    double error = pwm * 0.05 * TRANSFORM;  // 許容誤差[deg]
+
+    // 回頭前のモータカウント
+    int initialRightMotorCount = measurer.getRightCount();
+    int initialLeftMotorCount = measurer.getLeftCount();
+
+    PwmRotation.run();  // 左回頭を実行
+
+    // 回頭後に各モータが回転した角度
+    int rightMotorCount = abs(measurer.getRightCount() - initialRightMotorCount);
+    int leftMotorCount = abs(measurer.getLeftCount() - initialLeftMotorCount);
+
+    double actual = ((rightMotorCount + leftMotorCount) * TRANSFORM) / 2;
+
+    EXPECT_LE(expected, actual);
+    EXPECT_GE(expected + error, actual);
+  }
+
```

Comment:
```suggestion
// PWM値を0に設定して回頭するテスト
```
他のテストではテスト内容があるから、ここも。
名前はもっといい感じのあれば変えてください。